### PR TITLE
Linux-installing-a-deb-package-question-fix

### PR DESCRIPTION
**Pull request by abdullahtahir147**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/system-and-package-management/basic-package-management/installing-a-deb-package-from-the-terminal.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/linux/system-and-package-management/basic-package-management/installing-a-deb-package-from-the-terminal.md?ref=8033d20ede5372c021a5818cc985e953e62b1172)

File: linux/system-and-package-management/basic-package-management/installing-a-deb-package-from-the-terminal.md [hunk 1]
```diff
@@ -60,9 +60,10 @@ sudo dpkg -r PACKAGE_NAME
 The missing dependencies error thrown by `dpkg` can be fixed by:
 
 ```bash
-??? ??? ??? 
+??? ??? ??? ???
```
Comment (kapnobatai137) [inline linux/system-and-package-management/basic-package-management/installing-a-deb-package-from-the-terminal.md:63]: let's split this over two lines

```suggestion
??? ??? \
  ??? ??? 
```

File: linux/system-and-package-management/basic-package-management/installing-a-deb-package-from-the-terminal.md [hunk 2]
```diff
@@ -74,10 +75,10 @@ The missing dependencies error thrown by `dpkg` can be fixed by:
 
 ## Revision
 
-The usual syntax for installing a `deb` package is:
+The usual syntax for installing a `deb` package named `Deb_package_name` is:
 
 ```bash
-??? ??? ???
+sudo ??? ??? ???
```
Comment (kapnobatai137) [inline linux/system-and-package-management/basic-package-management/installing-a-deb-package-from-the-terminal.md:81]: same note here

```suggestion
sudo ??? \
  ??? ???
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**